src/main/java/org/opentripplanner/transit/raptor/api/path/PathLeg.java

```diff
@@ -159,6 +196,8 @@ default String asString() {
     }
 
     /**
+     * TODO TGR: REMOVE THIS IS VERY UC SPESIFIC
```

Comment:
What does this mean?

Reply:
I though the method was too specific or not generic enough. I wanted to remove it because I expected to not use it after the refactoring of the optimized transfer, but I can see that it is still in use in a couple of places so I will keep it. I am removing the TODO.